cmake/codecheck/rules/images_exist.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python3 -tt
+
+"""Checks that images loaded via g_gr->images() exist.
+
+Some images can't be checked, e.g. when they are being passed via a
+function.
+"""
+
+import os.path
+import re
+
+
+FIND_LITERALS = re.compile(r"""g_image_cache->get\("(.*?)"\)""")
+# We skip pointers and function calls here
+FIND_VARIABLE_NAMES = re.compile(
+    r"""g_image_cache->get\(([0-9a-zA-Z_-]*?)\)""")
```

Comment:
```suggestion
    r"""g_image_cache->get\(([0-9a-zA-Z_]+?)\)""")
```
Just a small tweak.

Reply:
Any particular reason you are removing the `-` character here? Changing `*` to `+` looks good.

Reply:
Variable names can't have `-`, so I don't know what it's doing there.

Reply:
Oops, let's get rid of it then.